app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -459,10 +465,15 @@ suspend fun openLink(
     val communityUrl = looksLikeCommunityUrl(parsedUrl)
 
     if (userUrl != null && (formatted || API.checkIfLemmyInstance(url))) {
-        val route = Route.ProfileFromUrlArgs.makeRoute(instance = userUrl.first, name = userUrl.second)
+        val route =
```

Comment:
How is this more readable?

Reply:
probably this is moving the line down for those who don't have widescreen monitors

Reply:
Was this done through a Linter? ktlint? ifso take over the extra rules have added in my other pr (expand search).